doc/nixpkgs-users-guide.rst

```diff
@@ -473,6 +473,14 @@ specific nix-shell You can either use an override as given above or
 simply install ``pkgs.haskellPackages.haskell-language-server`` instead
 of the top-level attribute ``pkgs.haskell-language-server``.
 
+How to use haskell-language-server in an editor with a GHC from nix-shell
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is solved fairly well with direnv:
+
+* Use `lorri <https://github.com/target/lorri>`__ to get information from the nix-shell to direnv
```

Comment:
Is maintained in the nix-community organization nowadays.

```suggestion
* Use `lorri <https://github.com/nix-community/lorri>`__ to get information from the nix-shell to direnv
```

Reply:
I updated https://github.com/direnv/direnv/wiki/Nix. But they really need to update the README in the target repo...

Reply:
Point taken!